Mos/zh-Hant-TW.lproj/Main.strings

```diff
@@ -1,113 +1,114 @@
 "Vs2-5P-BEI.title" = "平滑";
-"sKz-VE-Yfb.title" = "翻轉";
+"sKz-VE-Yfb.title" = "反轉";
 "89F-B4-NIY.title" = "應用程式";
 "mfa-4t-oRD.title" = "進階";
-"I5j-Bv-Aiy.title" = "需要用到輔助使用功能的權限";
-"FU4-wI-K8f.title" = "為了更精準地識別所設定的例外應用程式, 我們需要動態獲取運作中的視窗相關信息. 因此, 請允許我們獲取輔助使用功能的權限以保障該功能的正常運作";
-"Pw5-2q-7Fa.title" = "允許使用輔助使用功能";
+"I5j-Bv-Aiy.title" = "需要用到「輔助使用」權限";
+"FU4-wI-K8f.title" = "為更精確辨識出設定的例外應用程式，我們需要動態取得執行視窗的相關資訊。因此，請允許我們取得「輔助使用」功能的權限，讓這功能得以正常運作";
+"Pw5-2q-7Fa.title" = "允許使用「輔助使用」功能";
 "bga-wG-iim.title" = "例外";
-"Rgu-cb-5lp.title" = "例外的應用程式:";
+"Rgu-cb-5lp.title" = "例外應用程式：";
 "Ksp-g1-48g.title" = "白名單模式";
-"v26-RK-BtL.title" = "啟用白名單模式後, Mos 僅會對包含在例外列表中的應用程式啟用, 其他的則不受任何影響";
-"ThS-Rm-st2.title" = "沒有任何應用程式遭到不公對待";
-"jaq-Cb-dKM.title" = "你可以通過點擊下方的 \"+\" 來添加一個應用程式以設置其捲動行為";
+"v26-RK-BtL.title" = "啟用白名單模式後，Mos 只會對例外列表中的應用程式啟用，其他不受影響";
+"ThS-Rm-st2.title" = "沒有應用程式被區別對待";
+"jaq-Cb-dKM.title" = "按下方的「+」加入應用程式並設定其捲動行為";
 "R0d-21-g6W.title" = "執行中的應用程式";
-"dhF-95-kUJ.title" = "手動從 Finder 選擇";
-"Vp7-0n-aRd.title" = "手動輸入應用程式信息";
-"kUr-cl-KRm.title" = "應用程式名稱:";
-"Ck5-if-9RS.title" = "包名稱:";
+"dhF-95-kUJ.title" = "自行從 Finder 選擇";
+"Vp7-0n-aRd.title" = "自行輸入應用程式資訊";
+"kUr-cl-KRm.title" = "應用程式名稱：";
+"Ck5-if-9RS.title" = "套件名稱：";
 "Mdw-fZ-Nyy.title" = "取消";
-"2v4-AC-WG5.title" = "確認";
+"2v4-AC-WG5.title" = "確定";
 // Updates
 "EWu-DP-7Gj.title" = "更新";
-"vdI-K4-SBh.title" = "檢查更新:";
-"T9f-TL-GYj.title" = "獲得新版本";
-"5oM-oR-Dsq.title" = "當前版本";
+"vdI-K4-SBh.title" = "檢查更新：";
+"T9f-TL-GYj.title" = "尋找新版本";
+"5oM-oR-Dsq.title" = "目前版本";
 // About
 "RfV-Ow-3mF.title" = "關於";
-"5cx-xS-eRe.title" = "很慚愧, 就做了一點微小的工作";
-"yDf-dn-Mcz.title" = "主頁";
+"5cx-xS-eRe.title" = "抱歉，只幫到了一些些";
+"yDf-dn-Mcz.title" = "首頁";
 "99M-8D-HLN.title" = "GitHub";
 "AM8-cj-DnH.title" = "餵肥大咪";
-"aPi-cE-maK.title" = "通過 Paypal 捐贈";
-"FIh-fA-18k.title" = "歡迎界面";
-"265-WS-XEd.title" = "貢獻者們";
-"ZJY-Yg-Qev.title" = "貢獻者們";
+"aPi-cE-maK.title" = "透過 PayPal 捐贈";
+"FIh-fA-18k.title" = "歡迎介面";
+"265-WS-XEd.title" = "貢獻者名單";
+"ZJY-Yg-Qev.title" = "貢獻者名單";
 "Lhz-x5-X0I.title" = "譯者";
-"GWm-aK-UoV.title" = "貢獻者名目";
-"D0v-Oj-ekP.title" = "它有多胖?";
+"GWm-aK-UoV.title" = "貢獻者名單";
+"D0v-Oj-ekP.title" = "這🐱多胖？";
 
 // IntroductionWindow
 "VWv-6P-9Xp.title" = "上一步";
 "E9l-1I-6cD.title" = "下一步";
 "11f-uB-gEM.title" = "隆重介紹";
 "YHn-dn-A4n.title" = "Mos";
-"yFl-4N-WcD.title" = "你的滑鼠即將獲得順滑的滾動體驗, 抓穩你的滑鼠, 你僅僅需要調整幾個選項";
+"yFl-4N-WcD.title" = "你的滑鼠即將獲得順滑的滾動體驗，抓穩你的滑鼠！只有一些選項需要調整";
 "016-R7-U4x.title" = "你需要";
-"XP7-rO-U3i.title" = "檢查一些項目";
-"ia9-EG-sGg.title" = "某些應用程序(例如Logitech Options) 會對鼠標進行一些額外的處理\n但不需擔心, 只需要關閉他們的平滑滾動, 你仍然可以正常使用他們, 接下來的交給我們就好";
+"XP7-rO-U3i.title" = "檢查一些東西";
+"ia9-EG-sGg.title" = "某些應用程式（例如 Logitech Options）會對滑鼠做些額外處理\n但不需擔心，只要關閉這些程式的平滑捲動，您仍然可以正常使用，接下來的交給我們就好";
 "foc-QS-iys.title" = "我們需要";
 "Bbz-lG-LFI.title" = "您的授權";
-"Myd-Ze-ZPf.title" = "由於 macOS 的隱私控制限制, 我們只有在徵得輔助功能的訪問許可後才可以正常工作\n如果選項已存在, 取消選擇並再次勾選即可";
+"Myd-Ze-ZPf.title" = "由於 macOS 的隱私控制限制，我們只有在徵得「輔助功能」的存取權限後才得以正常運作\n如果已經勾過，取消選擇並再次勾選即可";
```

Comment:
From English translation ("Due to macOS's privacy restrictions, Mos requires access to Accessibility controls.\nIf the item already exists, just click and toggle the checkbox again."), it could be better to make it "由於 macOS 的隱私權限制，Mos 需要取得「輔助使用」的控制權限。\n如果已經勾過，取消選擇並重新勾選即可".